src/Build/Evaluation/Evaluator.cs

```diff
@@ -2547,7 +2547,16 @@ private void ThrowForImportedProjectWithSearchPathsNotFound(ProjectImportPathMat
                         importElement.Project.Replace(searchPathMatch.MsBuildPropertyFormat, extensionsPathPropValue),
                         ExpanderOptions.ExpandProperties, importElement.ProjectLocation);
 
-                relativeProjectPath = FileUtilities.MakeRelative(extensionsPathPropValue, importExpandedWithDefaultPath);
+                try
+                {
+                    relativeProjectPath = FileUtilities.MakeRelative(extensionsPathPropValue, importExpandedWithDefaultPath);
+                }
+                catch (ArgumentException ex)
```

Comment:
This is a good approach to work around the .NET Framework behavior, but I wonder if we can instead take advantage of the .NET Core behavior even in .NET Framework, using the `Microsoft.IO.Redist` package. Can you please try rewriting `FileUtilities.MakeRelative` to use `Microsoft.IO.Path.GetRelativePath` on .NET Framework, which shouldn't throw and should also be faster?

Reply:
@rainersigwald Thanks for your suggestions. But Path.GetRelativePath still throw exceptions when the path is empty.

Reply:
@rainersigwald  This fix is not only for .Net Framework.   Both .Net and .NET Framework, has the exception when extensionsPathPropValue is null or importExpandedWithDefaultPath is empty.  You can see the this from the test.